params/config.go

```diff
@@ -390,6 +392,54 @@ func (c *ChainConfig) String() string {
 	)
 }
 
+func (c *BorConfig) CalculateBackupMultiplier(number uint64) uint64 {
```

Comment:
In 'CalculateBackupMultiplier' and 'CalculatePeriod' you are repeating the same algorithm. It would be more clear if we would move it to a separated function.

Reply:
Will see how to merge them